charts/numaflow/templates/configmap.yaml

```diff
@@ -135,47 +138,11 @@ data:
             metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
             configReloaderImage: natsio/nats-server-config-reloader:0.7.0
             startCommand: nats-server
-          - version: 2.9.0
-            natsImage: nats:2.9.0
+          - version: 2.9.1
+            natsImage: nats:2.9.1
             metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
             configReloaderImage: natsio/nats-server-config-reloader:0.7.0
             startCommand: /nats-server
-          - version: 2.9.0-alpine
```

Comment:
What is the reason of removing these?

Reply:
Oh! Looks like I took these value from wrong place [here](https://github.com/numaproj/numaflow/blob/release-1.3/docs/operations/numaflow-controller-config.yaml), I should be taking value from [here](https://github.com/numaproj/numaflow/blob/release-1.3/config/base/controller-manager/numaflow-controller-config.yaml) instead, Let me update it